### PR TITLE
Feature/orchestrator-proto-refactor GetRuntime api

### PR DESCRIPTION
**Pull request by hoozecn**
refactor GetRuntime API

### CODE REVIEW

File: orchestrator/runtime/runtime.proto [hunk 1]
```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+// oap ~ Observability Analysis Platform
```
Comment (sfwn) [inline orchestrator/runtime/runtime.proto:3]: this comment is strange

**Review by sfwn — CHANGES_REQUESTED** [review]

File: orchestrator/runtime/runtime.proto [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+package erda.orchestrator.runtime;
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/erda-project/erda-proto-go/orchestrator/runtime/pb";
+
+message HealthzResponse {}
```
Comment (Effet) [inline orchestrator/runtime/runtime.proto:12]: use VoidRepsonse instead?

**Review by Effet — CHANGES_REQUESTED** [review]

File: orchestrator/runtime/runtime.proto (hunk 1 above)
Comment (hoozecn) [reply orchestrator/runtime/runtime.proto:3]: nice catch

File: orchestrator/runtime/runtime.proto (hunk 2 above)
Comment (hoozecn) [reply orchestrator/runtime/runtime.proto:11]: No prob

**Review by Effet — APPROVED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**